bigquery/README.md

```diff
@@ -4,6 +4,17 @@ This notifier pushes build data to a BigQuery instance.
 
 *Alpha Feature - Under Development*
 
+## Dataset and Table Setup
+
+The BQ notifier handles initializing datasets and tables automatically.
+Dataset identifiers in `bigquery.yaml` can refer to already existing or nonexistent datasets. 
```

Comment:
The file can be named whatever so prefer to say "BigQuery notifier config".

Reply:
+1